autonomy/cli/analyse.py

```diff
@@ -47,81 +47,59 @@ def analyse_group() -> None:
     """Analyse an agent service."""
 
 
-@analyse_group.group(name="abci")
-def abci_group() -> None:
-    """Analyse ABCI apps of an agent service."""
-
-
-@abci_group.command(name="generate-app-specs")
-@click.argument("app_class", type=str)
-@click.argument("output_file", type=click.Path())
+@analyse_group.command(name="fsm-specs")
```

Comment:
@dvilelaf @jmoreira-valory This refactor merges both generate and check commands into `autonomy analyse fsm-specs`

To check FSM specs for one package

`autonomy analyse fsm-specs --package PATH_TO_PACKAGE`

To check FSM specs for one package if FSM file does not exist

`autonomy analyse fsm-specs --package PATH_TO_PACKAGE --app-class ABCI_APP_NAME`

To update FSM specs for one package

`autonomy analyse fsm-specs --package PATH_TO_PACKAGE --app-class ABCI_APP_NAME --update`

To check FSM specs for all available packages

`autonomy analyse fsm-specs`

To specify packages directory 

`autonomy --registry-path PACKAGES_DIR analyse fsm-specs`

Do you have any suggestions to improve the UX for this command

Reply:
Where are you providing the input file  fsm_specification.yaml??  I only see PATH_TO_PACKAGE or ABCI_APP_NAME

Reply:
We automatically detect the file from the package path and the spec format

Reply:
https://github.com/valory-xyz/open-autonomy/pull/1504/files#diff-335efd8760dbf3e55acf9712ae4ce2285d87519d12e829b22cf481a6e56513baR79-R81